packages/sampler/stories/qa/Scroller.js

```diff
@@ -211,7 +229,14 @@ storiesOf('Scroller', module)
 	.add(
 		'With Many ExpandableList',
 		() => (
-			<Scroller focusableScrollbar={boolean('focusableScrollbar', Scroller, false)}>
+			<Scroller
+				focusableScrollbar={boolean('focusableScrollbar', Config)}
+				horizontalScrollbar={select('horizontalScrollbar', prop.scrollbarOption, Config)}
+				noScrollByWheel={boolean('noScrollByWheel', Config)}
```

Comment:
Why don't you add `onScrollStart` and `onScrollStop`? We don't usually add `onScroll` due to performance issue.

Reply:
we could do that but I'd like to do the work next time. First of all, the current work is only for adding extra knobs but the work you mentioned is about action. I think it would be better to separate them. Second, I'm not used to using storybook action. Please let me know your thoughts on this.

Reply:
Ok. Let's do it as a separate issue.